uint16 id
float32 pos_estimate
float32 vel_estimate
float32  torque_estimate